keyboards/mountainmechdesigns/teton_78/readme.md

```diff
@@ -0,0 +1,30 @@
+# teton_78
+
+![teton_78](https://imgur.com/IwQgOy3.jpg)
+
+A 68 XT designed by me (Bennett Balogh)! Sold at mountainmechdesigns.com
+
+* Keyboard Maintainer: [Bennett Balogh (https://github.com/AwesomeBalogh)
+* Hardware Supported:Atmega34u4 for Teton 78s' Hotswap and Solder PCBs
+* Hardware Availability: *N/A questions direct to Mountain Mech Designs discord: https://discord.gg/h9dMwRNfVy*
+
+Make example for this keyboard (after setting up your build environment):
+
+    make teton_78:default
+    make teton_78:via
+    qmk compile -kb teton_78/ -km default
+
+Flashing example for this keyboard:
+
+    make teton_78:default:flash
+    make teton_78:via:flash
```

Comment:
A link to the product page would be most preferable here.
```suggestion
* Keyboard Maintainer: [Bennett Balogh (https://github.com/AwesomeBalogh)
* Hardware Supported: Atmega34u4 for Teton 78s' Hotswap and Solder PCBs
* Hardware Availability: [Teton 78 Group Buy](https://www.mountainmechdesigns.com/shop/p/product-5-fb9r2)

Make example for this keyboard (after setting up your build environment):

    make mountainmechdesigns/teton_78:default

Flashing example for this keyboard:

    make mountainmechdesigns/teton_78:default:flash
```

Reply:
Updated readme :)